includes/admin/upgrades/upgrade-functions.php

```diff
@@ -2758,6 +2762,48 @@ function give_v221_upgrades() {
 	$wpdb->query( "ALTER TABLE $wpdb->donors MODIFY email varchar(255) NOT NULL" );
 }
 
+/**
+ * Version 2.3.0 automatic updates
+ *
+ * @since 2.3.0
+ */
+function give_v230_upgrades() {
+
+	$options_key = array(
+		'give_temp_delete_form_ids', // delete import donor
+		'give_temp_delete_donation_ids', // delete import donor
+		'give_temp_delete_step', // delete import donor
+		'give_temp_delete_donor_ids', // delete import donor
+		'give_temp_delete_step_on', // delete import donor
+		'give_temp_delete_donation_ids', // delete test donor
+		'give_temp_delete_donor_ids', // delete test donor
+		'give_temp_delete_step', // delete test donor
+		'give_temp_delete_step_on', // delete test donor
+		'give_temp_delete_test_ids', // delete test donations
+		'give_temp_all_payments_data', // delete all stats
+		'give_recount_all_total', // delete all stats
+		'give_temp_recount_all_stats', // delete all stats
+		'give_temp_payment_items', // delete all stats
+		'give_temp_form_ids', // delete all stats
+		'give_temp_processed_payments', // delete all stats
+		'give_temp_recount_form_stats', // delete form stats
+		'give_temp_recount_earnings', // recount income
+		'give_recount_earnings_total', // recount income
+		'give_temp_reset_ids', // reset stats
+	);
+
+	$options_key = '\'' . implode( "','", $options_key ) . '\'';
+
+	global $wpdb;
+
+	/**
+	 * Update  1
+	 *
+	 * Change column length
+	 */
+	$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name IN ( {$options_key} )" );
```

Comment:
@raftaar1191 `IN` query will be slow, so use multiple `option_name ={$option_name}` query

Reply:
@ravinderk fix now the query will be like

```
string(920) "DELETE FROM wp_options WHERE option_name = 'give_temp_delete_form_ids' OR option_name = 'give_temp_delete_donation_ids' OR option_name = 'give_temp_delete_step' OR option_name = 'give_temp_delete_donor_ids' OR option_name = 'give_temp_delete_step_on' OR option_name = 'give_temp_delete_donation_ids' OR option_name = 'give_temp_delete_donor_ids' OR option_name = 'give_temp_delete_step' OR option_name = 'give_temp_delete_step_on' OR option_name = 'give_temp_delete_test_ids' OR option_name = 'give_temp_all_payments_data' OR option_name = 'give_recount_all_total' OR option_name = 'give_temp_recount_all_stats' OR option_name = 'give_temp_payment_items' OR option_name = 'give_temp_form_ids' OR option_name = 'give_temp_processed_payments' OR option_name = 'give_temp_recount_form_stats' OR option_name = 'give_temp_recount_earnings' OR option_name = 'give_recount_earnings_total' OR option_name = 'give_temp_reset_ids'"

```

Reply:
@raftaar1191 It is still the same as `IN` query. I will take care fo that